cumulus_etl/etl/studies/covid_symptom/covid_tasks.py

```diff
@@ -58,42 +57,13 @@ def is_ed_docref(docref):
     return any(is_ed_coding(x) for x in codings)
 
 
-class CovidSymptomNlpResultsTask(tasks.EtlTask):
-    """Covid Symptom study task, to generate symptom lists from ED notes using NLP"""
+class BaseCovidSymptomNlpResultsTask(tasks.BaseNlpTask):
+    """Covid Symptom study task, to generate symptom lists from ED notes using cTAKES + a polarity check"""
```

Comment:
A lot of this PR is refactoring, sorry.

I make a new BaseNlpTask for all NLP jobs (shared by the hugging face test task and the covid tasks) PLUS a base covid symptoms test here that both the (nearly-identical) covid tasks inherit from).

And the tests when you get to them are also refactored a bit - I added a new shared BaseEtlTest class for all the tests that care about running a full ETL and some refactoring to handle the changes from dropping the covid tests from the default list of ETL jobs (keep reading for that change).